### DISCUSSION TITLE
Unable to do a multiline run command

### DISCUSSION THREAD

**Discussion by phenry20** [community/community#25878] [Actions]
<p>Hi, </p>
<p>Whenever I try to do multi-line run commands I keep getting the following error: </p>
<pre><code>### ERRORED 20:52:01Z

- Your workflow file was invalid: The pipeline is not valid. .github/workflows/build.yml (Line: 18, Col: 7): There's not enough info to determine what you meant. Add one of these properties: run, shell, uses, with, working-directory,.github/workflows/build.yml (Line: 22, Col: 7): Unexpected value 'run'
</code></pre>
<p>Below is my .yml file: </p>
<pre><code>name: build
on:
  push:
    branches-ignore:
      - 'master'
    tags-ignore:
      - v*
jobs:
  build_and_release:
    runs-on: ubuntu-latest
    steps:
    - uses: actions/checkout@v1

    - name: Build
      run: docker build -t $CONTAINER_RELEASE_IMAGE:test .
      env:
        REGISTRY: registry.gitlab.com
        CONTAINER_RELEASE_IMAGE: $REGISTRY/team/group/project

    - name: Release
    - uses: actions/setup-node@v1
      with:
        node-version: '10.x'
      run: |
        npm install -g semantic-release@15.13.24 @semantic-release/exec @semantic-release/commit-analyzer
        docker login -u registryuser -p $CI_REGISTRY_TOKEN $REGISTRY
        docker push $CONTAINER_RELEASE_IMAGE:test
</code></pre>
<p>If I make the run command a single line everything works, but as soon as I attempt multi line commands it always fails no matter what I do. Am I doing something wrong or is this a bug?</p>
<p>Thanks in advance for your replies</p>

**Comment by mhart** [comment]
<pre><code>- name: Release
    - uses: actions/setup-node@v1
</code></pre>
<p>Fix that ^ and it should work</p>

**Comment by mhart** [comment, marked as answer]
<p>To be more specific – that’s actually two steps there, and one only has a name. I think it should be:</p>
<pre><code>- name: Release
      uses: actions/setup-node@v1
</code></pre>

**Comment by phenry20** [comment]
<p>Nevermind I figured it out. It seems like actions and commands can’t be run in the same step. </p>

**Comment by phenry20** [comment]
<p>Haha looks like we replied at the same time. Yes I figured that out after looking at more examples. </p>
<p>Thanks so much! :) </p>
